docs/UserGuide.md

```diff
@@ -1,17 +1,40 @@
 # BankWithUs User Guide
 
 --------------------------------------------------------------------------------------------------------------------
-
-## Quick start guide
+## <span style="color:#CC0000">Table of contents:</span>
+* [Quick-start guide](#quick-start)
+* [Features](#features)
+    * [Help message](#help-message)
+    * [Add new account](#add-account)
```

Comment:
[Switch Account]
[Delete Account]
[View Account]
[Deposit]
[Withdraw]
[Transaction History]
[Save Goal]
[Withdraw Limit]
[Check Withdraw Limit]
[Delete Transaction History]
[Exiting the program]
[Frequently Asked Questions (FAQ)]

These links did not seem to work when I tried to click on them

Reply:
Now should be working